README.md

```diff
@@ -85,9 +85,9 @@ then you can type `pipenv install` instead. Once these commands complete
 successfully, that's all you you have to do to install GatorGrader! It is worth
 noting that if you only plan to use GatorGrader with the
 [GatorGradle](https://github.com/GatorEducator/gatorgradle) then there is a
-[java sample laboratory
-assignment](https://github.com/GatorEducator/java-starter) that you can
-try &mdash; it does not require you to complete these steps and instead it will
+[Java sample laboratory
+assignment](https://github.com/GatorEducator/java-starter), [LaTeX sample laboratory assignment](https://github.com/GatorEducator/latex-assignment-starter), & [HTML with CSS sample laboratory assignment](https://github.com/GatorEducator/html-css-assignment-starter) that you can
```

Comment:
I would suggest leaving off the "sample laboratory assignment" on every one, and mention it elsewhere, to make this list smaller.

Reply:
New commit should resolve your concerns @Michionlion